docker-compose.yml

```diff
@@ -1,6 +1,40 @@
-app:
-    build: .
-    volumes:
-        - .:/app/
-    ports:
-        - "8000:80"
+version: '2'
+services:
+    db:
+        image: mysql:5.5
+        environment:
+            - MYSQL_ROOT_PASSWORD=root
+            - MYSQL_DATABASE=premis_local
+
+    base:
+        build: .
+        volumes:
+            - .:/app/
+
+    app:
+        extends:
+            service: base
+        depends_on:
+            - db
+        environment:
+            - DJANGO_SETTINGS_MODULE=tests.settings
+            - PES_BACKEND=mysql
+        command: /appstart.sh
```

Comment:
Does the command need to read `./appstart.sh`? It errored for me the way it was.